src/client/ShopContext.ts

```diff
@@ -203,7 +203,15 @@ export class ShopContext {
     const path = '/api/v2/order/get_order_detail'
     const resp = await this.ax.get(path, {
       params: {
-        order_sn_list: orderSn.join()
+        order_sn_list: orderSn.join(),
+        response_optional_fields:[
+          'buyer_username',
+          'total_amount',
+          'estimated_shipping_fee',
+          'item_list',
+          'payment_method',
+          'shipping_carrier',
+        ].join(),
```

Comment:
Shouldn't be hardcoded. Rather it should be provided as a query parameters.

```ts
type OrderDetailOptionalFields = 'buyer_username' | 'total_amount' | 'estimated_shipping_fee' | 'item_list' | 'payment_method' | 'shipping_carrier'
```

Then in api parameter it should get.

```ts
getOrderDetail(orderSn: string[], optionalFields: OrderDetailOptionalFields[]) { ... }
```